packages/ember/tests/dummy/app/templates/docs/coordination.md

```diff
@@ -0,0 +1,68 @@
+# Coordination
+
+For many simple uses of milestones, the details of the coordination between them can be glossed over. If you're skimming these guides to get a high level overview of the library, you may even want to skip this section and come back when you find yourself needing more details.
+
+However, if you're dealing with hairy concurrency situations with multiple different logical threads of execution under test, understanding how different sets of milestones interact is key to ensuring you have the degreee of control you need and that your tests are deterministic.
+
+## Milestone Coordinators
+
+Each time you call `activateMilestones`, it creates a {{#link-to 'docs.api.item' 'modules/@milestones/core~MilestoneCoordinator'}}`MilestoneCoordinator`{{/link-to}} instance.
+
+When a milestone is reached, it checks to see if there's an active coordinator for either its ID or one of its tags. If so, it pauses until that coordinator tells it how to continue. When you call `advanceTo` with a milestone key, it locates the corresponding coordinator and waits until it's ready with the pending milestone, and then passes your instruction back to the paused milestone for how to proceed.
+
+A given milestone coordinator can only be paused at or advancing to a single milestone at a time. As mentioned in {{#link-to 'docs.interacting-with-milestones'}}Interacting with Milestones{{/link-to}}, if you're currently advancing to a particular milestone, all others belonging to the same coordinator that are reached along the way will automatically behave as if you called `.continue()` on them. This prevents a deadlock scenario where your tests are waiting to reach one milestone, but your application code is waiting to hear how to proceed with an earlier one.
```

Comment:
> A given milestone coordinator can only be paused at or advancing to a single milestone at a time.

Something about this is hard to read

Reply:
Hmm. Maybe the paragraph reads better if I just drop that sentence completely? The next one describes the actual behavior that devs need to care about.